projects/demo/src/modules/components/primitive-textfield/examples/2/index.ts

```diff
@@ -0,0 +1,66 @@
+import {
+    ChangeDetectorRef,
+    Component,
+    Inject,
+    Optional,
+    Self,
+    ViewChild,
+} from '@angular/core';
+import {NgControl} from '@angular/forms';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {AbstractTuiControl, TuiNativeFocusableElement} from '@taiga-ui/cdk';
+import {
+    TUI_PRIMITIVE_TEXTFIELD_DEFAULT_OPTIONS,
+    TUI_PRIMITIVE_TEXTFIELD_OPTIONS,
+    TuiPrimitiveTextfieldComponent,
+} from '@taiga-ui/core';
+
+@Component({
+    selector: 'tui-primitive-textfield-example-2',
+    templateUrl: './index.html',
+    changeDetection,
+    encapsulation,
+    providers: [
+        {
+            provide: TUI_PRIMITIVE_TEXTFIELD_OPTIONS, // <-- You are looking for this token
+            useValue: {
+                ...TUI_PRIMITIVE_TEXTFIELD_DEFAULT_OPTIONS,
+                iconCleaner: 'tuiIconChevronUp',
+            },
+        },
+    ],
+})
+export class TuiPrimitiveTextfieldExample2 extends AbstractTuiControl<string> {
+    @ViewChild(TuiPrimitiveTextfieldComponent)
+    private readonly textfield?: TuiPrimitiveTextfieldComponent;
+
+    constructor(
+        @Optional() @Self() @Inject(NgControl) control: NgControl | null,
```

Comment:
nit: need use `prettier` here

Reply:
I also noticed that it didn't take care of everything in these commits as it usually would but thought that it's due to configuration change. I'll look into it

Reply:
it would be great if you find a bug in our prettier configuration

Reply:
@splincode, I fixed my local husky config and that fixed a lot of issues with the pre-commit hook but I still can't figure out why this particular line of code is not being taken care of, except that it's only 73 chars long

Reply:
Perhaps it's correct from prettier point of view? :)

Reply:
oh)) I see, this is the correct line for prettier

![image](https://user-images.githubusercontent.com/12021443/150946197-d7cf17eb-9664-41e5-8e40-74e16211f52f.png)

please, add manually break line for decorators))

Reply:
@mullinsmikey also, please, tell me, what did you have to do with the husky to make it work?)

Reply:
The point of prettier is to never add manual breaks and accept prettier as objective reality and a higher power :D

Reply:
prettier does breaks only if the line exceeds the maximum length of 80 characters for typescript code )) it does not analyze what kind of construct is used in the code, decorators or something else.

Reply:
@splincode, sure.
When you use nvm sometimes npx is not present in your path. Usually this is not a problem, but GUI git clients may produce an error like "npx command not found". Here's an [official way](https://typicode.github.io/husky/#/?id=command-not-found) to fix this on your machine, but there's a [quick&dirty way](https://github.com/typicode/husky/issues/912#issuecomment-817522060) to persist this behavior in the repo.

Reply:
@mullinsmikey could you add please `quick@dirty` or `official` way to fix it in another PR?